.github/workflows/main.yaml

```diff
@@ -0,0 +1,87 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - "*"  # run for branches
+    tags:
+      - "*"  # run for tags
+  pull_request:
+    branches:
+      - "*"  # run for branches
+    tags:
+      - "*"  # run for tags
+
+jobs:
+  test:
+    defaults:
+      run:
+        working-directory: go/src/github.com/microservices-demo/user
+    runs-on: ubuntu-latest
+    env:
+      GROUP: weaveworksdemos
+      COMMIT: ${{ github.sha }}
+      REPO: user
+      GO_VERSION: 1.7.5
+      GOPATH: /home/runner/work/user/user/go
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+          path: go/src/github.com/microservices-demo/user
+
+      - uses: actions/setup-go@v1
+        with:
+          go-version: ${{ env.GO_VERSION }}
+
+      - name: Setup PATH
+        run: echo "::add-path::${GOPATH}/bin"
+
+      - name: Install dependencies
+        run: make deps && go get -v github.com/mattn/goveralls
+
+      - name: Install/Downgrade MongoDB to version 3.6
+        run: |
+          wget -qO - https://www.mongodb.org/static/pgp/server-3.6.asc | sudo apt-key add -
+          echo "deb [ arch=amd64 ] https://repo.mongodb.org/apt/ubuntu bionic/mongodb-org/3.6 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-3.6.list
+          sudo apt-get update
+          sudo apt-get install -y --allow-downgrades mongodb-org=3.6.20 mongodb-org-server=3.6.20 mongodb-org-shell=3.6.20 mongodb-org-mongos=3.6.20 mongodb-org-tools=3.6.20 || echo "Error is misleading, it was successfully installed"
+
+      - name: Unit Tests
+        run: glide novendor| xargs go test -v
+
+      - name: Create cover profile
+        run: make coverprofile
+
+      - name: Run Docker Test
+        run: make dockertest
+
+      - name: Submit Coveralls
+        env:
+          COVERALLS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: goveralls -coverprofile=cover.profile -service=github
+
+      # Push to dockerhub
+      - name: Push user to Docker Hub
+        uses: docker/build-push-action@v1
+        if: startsWith(github.ref, 'refs/tags/v') || github.ref == 'refs/heads/master'
+        with:
+          username: ${{ secrets.DOCKER_USER }}
+          password: ${{ secrets.DOCKER_PASS }}
+          dockerfile: docker/user/Dockerfile
```

Comment:
You need to update the dockerfile to `go/src/github.com/microservices-demo/user/docker/user/Dockerfile-release`

Reply:
I fixed dockerfile name, I think the paths should be working as is, because in the `defaults` section - https://github.com/microservices-demo/user/pull/63/files#diff-295c250d2ddc435b0eafc807f0b3cac9R17-R19, I am changing working directory to `go/src/github.com/microservices-demo/user/`.

Reply:
My bad, it looks like steps with `uses` are not respecting `working-directory`, good catch @elieser1101 ! Thanks!